app/src/main/java/org/jellyfin/androidtv/details/EpisodeDetailsFragment.kt

```diff
@@ -0,0 +1,59 @@
+package org.jellyfin.androidtv.details
+
+import android.graphics.drawable.BitmapDrawable
+import android.os.Bundle
+import android.util.Log
+import androidx.leanback.widget.*
+import org.jellyfin.androidtv.model.itemtypes.Episode
+
+class EpisodeDetailsFragment(private val data: Episode) : BaseDetailsFragment() {
+
+	override fun onCreate(savedInstanceState: Bundle?) {
+		super.onCreate(savedInstanceState)
+
+		buildDetails()
+	}
+
+	private fun buildDetails() {
+
+		Log.i("EpisodeDetailsFragment", data.name)
+
+		val selector = ClassPresenterSelector().apply {
+			// Attach your media item details presenter to the row presenter:
+
+			val detailsDescriptionPresenter = DetailsDescriptionPresenter(title = data.name, subtitle = "TODO", body = data.overview)
+
+			if (data.primaryImage != null) {
```

Comment:
I'd put the if statement inline so you don't need to repeat the same code twice:
```kotlin
FullWidthDetailsOverviewRowPresenter(detailsDescriptionPresenter, data.primaryImage?.let { DetailsOverviewLogoPresenter() })
```